mappings/net/minecraft/block/entity/SpawnerBlockEntityBehavior.mapping

```diff
@@ -1,7 +1,10 @@
 CLASS net/minecraft/class_88 net/minecraft/block/entity/SpawnerBlockEntityBehavior
 	FIELD field_199 defaultMob Ljava/lang/String;
-	METHOD method_181 (Lnet/minecraft/class_1405;)V
+	METHOD method_176 deserialize (Lnet/minecraft/class_1405;)V
+	METHOD method_180 handleStatus (I)Z
```

Comment:
this doesnt feel like a yarn name

Reply:
there's a method in `SpawnerMinecartEntity` called `handleStatus` that calls the one in `SpawnerBlockEntityBehavior`
hence I named it so

Reply:
ah ok. fair decision